types/_collections/tags.d.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+declare global {
+  export interface TagsItem {
```

Comment:
should this extend TagsDataItem similar to how AuthorsItem extends AuthorsDataItem?

I think maybe the only shared properties are title and description but maybe it's good to do for consitency?

Reply:
I believe the issue is that I can not overwrite the typing of a property. For tagsData, the `description` is optional, where as in the tagsCollection it's mandatory.